lib/foreman_remote_execution/engine.rb

```diff
@@ -138,6 +138,7 @@ class Engine < ::Rails::Engine
         end
 
         extend_rabl_template 'api/v2/smart_proxies/main', 'api/v2/smart_proxies/pubkey'
+        extend_rabl_template 'api/v2/interfaces/main', 'api/v2/remote_execution_features/host_extensions'
```

Comment:
It is actually an extension of interface so the name is slightly misleading.

Reply:
renamed to `interface_extensions`